plugins/background_fetch/rules.h

```diff
@@ -58,8 +58,8 @@ class BgFetchRule
 
   // Main evaluation entry point.
   bool bgFetchAllowed(TSHttpTxn txnp) const;
+  bool check_field_configured(TSHttpTxn txnp) const;
 
-private:
```

Comment:
If these are being made public, shouldn't the initial _ be dropped.  Also, if they are public can they be made const, like:
const bool exclude;
const char * const field;
const char * const value;

?